docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+---
+sidebar_position: 4   
+description: Changing the suit's name and description 2 
+---
+
+import './customhover.css';
+import Tooltip from './tooltip';
+
+
+# Adding Localisation Overrides
+
+*In this tutorial, you’ll learn how to modify suit names and descriptions in the game using* 
+> <Tooltip text={<><strong>Localisation Overrides</strong><span className="icon-alert"></span></>} tooltip="A way to replace or modify in-game text, like names or descriptions, with your own custom text." />. 
+
+You'll learn how to:
+- locate the necessary files, 
+- identify and convert *"strings"*, 
+- update the manifest with your custom text. 
+
+>By the end, you'll have a fully customized suit name and description in the game.
+
+*When necessary, there will be a link to the ***advanced search section*** to change any other text as well*
+
+
+<details>
+<summary>**About Tooltips**</summary>
```

Comment:
I feel like this whole section can be condensed to "Tooltips are included throughout the article. You can hover over them for a quick explanation."